Gyor Plusz.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "67b93a38-5182-45e5-ab35-3b95764f6979",
+	"label": "Győr Plusz",
+	"creator": "homope",
+	"target": "^https?://(www\\.)?gyorplusz\\.hu",
+	"minVersion": "6.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2025-12-10 12:15:00"
```

Comment:
A translator from the future? (I really highly recommend using Scaffold, Tools -> Translator Editor, rather than editing this stuff by hand.)